packages/synapse-react-client/src/components/CardContainer/CardContainer.tsx

```diff
@@ -158,7 +172,13 @@ function _CardContainer(props: CardContainerProps) {
     <>
       <Box
         role="list"
-        sx={isReleaseCardMediumList ? releaseCardMediumListSx : defaultListSx}
+        sx={
+          isReleaseCardMediumList
+            ? releaseCardMediumListSx
+            : multiCardList && isMultipleCards
+            ? multiCardListSx
+            : defaultListSx
```

Comment:
Instead of multiple ternary operators, could you write a series of if/else statements above to figure out this value?
I realize this is more verbose, but I think it improves readability and can be easily extended.
Something like
```
let cardListSx = defaultListSx
if (isReleaseCardMediumList) {
  cardListSx = releaseCardMediumListSx
} else if (multiCardList && isMultipleCards) {
  cardListSx = multiCardListSx
}
```

Reply:
@nickgros / @sagely1 , what are your thoughts/opinions?

Reply:
I agree! Using if/else statements would make it a lot easier to extend.